DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1,8 +1,6 @@
 class DXREvents extends DXRActorsBase;
 
-var transient #var PlayerPawn  _player;
-var transient bool died;
-
+var bool died;
```

Comment:
I need to test how this works with HX, might need to store the player that died, or even an array of players considering many could die in the same explosion